### DISCUSSION TITLE
Could we get a build for 2.7 with the OpenJ9 support?

### DISCUSSION THREAD

**Discussion by ajruckman** [async-profiler/async-profiler#547] [Ideas]
Would really like to start using it with another project :)

**Comment by apangin** [comment]
The nearest release will include OpenJ9 support.
It is now in testing stage.

Meanwhile you may try the Early Access build:
https://profiler.tools/builds/async-profiler-2.7-ea-linux-x64.tar.gz

Since OpenJ9 is compiled without frame pointers, use `--cstack dwarf` to see the native frames in the profile.

Looking forward to any feedback.

**Reply by apangin** [reply]
Release 2.7 has been [published](https://github.com/jvm-profiling-tools/async-profiler/releases/tag/v2.7).
